File: Service_eHospital_DongNai_A/Implement/CUS_ProductsService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CUS_ProductsService : BaseService<CUS_Products, ICUS_ProductsRepository>
    , ICUS_ProductsService
    {
    private readonly ICUS_ProductsRepository _CUS_ProductsRepository;
    public CUS_ProductsService(ICUS_ProductsRepository CUS_ProductsRepository) : base(CUS_ProductsRepository)
    {
    _CUS_ProductsRepository = CUS_ProductsRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/CUS_Products.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CUS_Products } from './CUS_Products.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CUS_ProductsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Code', 'Name', 'Unit', 'Quantity', 'Price', 'Total', 'Action', 'Fkey', 'Extra1', 'Extra2', 'Remark'];

    List: CUS_Products[] | undefined;
    ListFilter: CUS_Products[] | undefined;
    FormData!: CUS_Products;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CUS_Products";
    }
}

